configure.ac

```diff
@@ -1549,7 +1549,7 @@ dnl ------------------------
 dnl TCP_MD5SIG socket option
 dnl ------------------------
 
-AC_CHECK_HEADER([netinet/tcp.h], 
```

Comment:
Please remove whitespace changes from this commit

Reply:
Fixed.